Translations for logbook/history panels

emlove opened the pull request:
## Description
UI translation for the logbook/history panels.

### Before merging
- [x] Rebase on master for new localize function

emlove committed:
Translations for logbook/history panels

andrey-git commented on panels/history/ha-panel-history.html:
```diff
@@ -61,22 +62,22 @@
-              <paper-item>1 day</paper-item>
-              <paper-item>3 days</paper-item>
-              <paper-item>1 week</paper-item>
+              <paper-item>[[haLocalize('ui.panel.history.periods', 'day', 'count', 1)]]</paper-item>
```
Shouldn't this be `haLocalize('ui.panel.history.periods.day', 'count', 1)` ?

armills replied:
I agree.

I'm going to open a separate PR and just refactor away the `haLocalize` wrapper. I think I had catered it mostly to the original translation use-case, and most of the method has already been eliminated. At this point it's just more confusing/complicated than using the Polymer `localize` directly.

emlove committed:
Merge branch 'master' into logbook-history-translations

emlove committed:
Switch to base localize function

emlove committed:
Move periods to generic durations keys

These are going to end up part of a duration class of keys that will be
more widely used.
